src/NetworkInterface.cpp

```diff
@@ -3229,6 +3263,25 @@ static bool find_mac_by_name(GenericHashEntry *h, void *user_data, bool *matched
 
 /* **************************************************** */
 
+
+static bool find_arp_stats_element_by_macs_name(GenericHashEntry *h, void *user_data, bool *matched) {
+
+  struct arp_stats_element_find_info *info = (struct arp_stats_element_find_info*)user_data;
+
+  ArpStatsMatrixElement *e = (ArpStatsMatrixElement*)h;
+
+  if( (info->elem == NULL) && !info->elem->equal(e->getSourceMac(), e->getDestinationMac()) ){
```

Comment:
code inside this is IF is never reached and is going to crash with a SEGFAULT when info->elem is null